core/state/statedb.go

```diff
@@ -349,6 +350,34 @@ func (sw *StateDB) ApplyMVWriteSet(writes []blockstm.WriteDescriptor) {
 	}
 }
 
+// get readMap Dump of format: "TxIdx, Inc, Path, Read"
+func (s *StateDB) GetReadMapDump() string {
+	str := ""
+	for _, val := range s.MVReadList() {
+		// fmt.SprintF is expensive, but can be sued for debugging
+		str += fmt.Sprintf("%v", s.txIndex) + ", " + fmt.Sprintf("%v", s.incarnation) + ", " + fmt.Sprintf("%v", val.V.TxnIndex) + ", " + fmt.Sprintf("%v", val.V.Incarnation) + ", " + hex.EncodeToString(val.Path) + ", " + "Read\n"
+	}
+
+	return str
+}
+
+// get writeMap Dump of format: "TxIdx, Inc, Path, Write"
+func (s *StateDB) GetWriteMapDump() string {
+	str := ""
```

Comment:
Do we need this API fast or it is not critical? If it should be performant, then
1. remove all fmt.Sprintf in favor of 1 fmt.Sprintf with 1 format string
2. try string.Builder
3. maybe it worth it to stick with []byte rather then strings

Reply:
Right, so we will be using this only to debug, that was the reason I went ahead with fmt.Sprintf.
And about using []byte, we wanted to make the result readable, therefore using strings.

Reply:
However it's possible to convert []byte as a very last step before sending data to a user.

Reply:
Ohh, okay. I will do that change. Thank you.